src/ORM/LazyLoadEntityTrait.php

```diff
@@ -48,7 +49,7 @@ public function &get($property)
      */
     protected function &_parentGet($property)
     {
-        return parent::get($property);
+        return Entity::get($property);
```

Comment:
Now this really runs? I am baffled.

Reply:
I know. When I asked about it my mind was blown. I'm happy to still be learning :)